packages/cli/src/bin/pipcook-daemon.ts

```diff
@@ -36,7 +37,8 @@ async function start(): Promise<void> {
     if (message.event === 'ready') {
       daemon.disconnect();
       daemon.unref();
-      logger.success(`Pipcook service started on ${message.data.listen}. To open pipboard, please visit https://pipboard.vercel.app`);
+      const serverInfo = (message.data as any).listen ?? `${message.data.host}:${message.data.port}`;
```

Comment:
We should split this expression into an `if-else` clause, and add some comments to describe the reason to check.